.github/workflows/ci.yml

```diff
@@ -66,10 +66,13 @@ jobs:
         path: ${{ env.TEST_RESULTS_DIR }}
 
   e2e:
-    name: e2e tests
+    name: e2e tests (consul-image=${{ matrix.consul-image }})
+    strategy:
+      matrix:
+        consul-image: ['hashicorp/consul:1.11.2', 'hashicorp/consul-enterprise:1.11.2-ent']
```

Comment:
TIL 🤩 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategy